### PR TITLE
Use shortcode for note in Kops glossary entry

### PR DESCRIPTION
**Pull request by sftim**
- Use shortcode for note in Kops glossary entry
- also change “terraform” to “Terraform”

### CODE REVIEW

File: content/en/docs/reference/glossary/kops.md [hunk 1]
```diff
@@ -4,25 +4,29 @@ id: kops
 date: 2018-04-12
 full_link: /docs/getting-started-guides/kops/
 short_description: >
-  A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters. *NOTE&#58; Officially supports AWS only, with GCE and VMware vSphere in alpha*.
+  A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters.
```
Comment (DanyC97) [inline content/en/docs/reference/glossary/kops.md]: nit: i think we can take out `you`

File: content/en/docs/reference/glossary/kops.md [hunk 2]
```diff
@@ -4,25 +4,29 @@ id: kops
...
 
 aka: 
 tags:
 - tool
 - operation
 ---
- A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters. *NOTE&#58; Officially supports AWS only, with GCE and VMware vSphere in alpha*.
+ A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters.
```
Comment (DanyC97) [inline content/en/docs/reference/glossary/kops.md]: likewise here

**Comment by DanyC97** [comment]
minor nits, apart from looks good

File: content/en/docs/reference/glossary/kops.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/glossary/kops.md]: Per the style guide, I'm happy to address the reader as “you”.
Compare to:
> kubeadm helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices

from https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/

Comment (DanyC97) [reply content/en/docs/reference/glossary/kops.md]: okay ... is just didn't feel right to me...

**Pull request merged**